src/honeycomb-options.ts

```diff
@@ -61,6 +63,12 @@ export interface HoneycombOptions extends Partial<NodeSDKConfiguration> {
   /** The OTLP protocol used to send telemetry to Honeycomb. The default is 'http/protobuf'. */
   protocol?: OtlpProtocol;
 
+  /** Number of milliseconds for the metric reader to initiate metric collection. Defaults to 60000. Must be greater than metricsTimeout. */
+  metricsInterval?: number;
```

Comment:
Now that I think about it, adding this to the `HoneycombOptions` interface might signal to users that they can configure this in the code itself so it might be surprising behaviour that setting this option doesn't work unless they do it through environment variables. There's three solutions I can think of:
1. We have a separate interface for `ComputedHoneycombOptions` so that we can have options that aren't directly configurable by the user. This is quite a bit of a refactor so I'm not sure it's the best option right now.
2. Instead of getting these values from `HoneycombOptions`, we directly call `getMetricsInterval` and `getMetricsTimeout` where these values get set in `PeriodicExportingMetricReader`.
3. We allow these values to be configurable in code 

I think its also okay to leave this as is for this PR since we're still in beta and address it in a subsequent PR / release. Lemme know what you think!